harness/tests/fixtures/api_responses.py

```diff
@@ -53,6 +53,21 @@ def sample_get_model_versions() -> bindings.v1GetModelVersionsResponse:
         return resp
 
 
+def sample_login() -> bindings.v1LoginResponse:
+    resp = bindings.v1LoginResponse(token="fake-login-token", user=sample_get_user().user)
+    return resp
+
+
+def sample_get_user() -> bindings.v1GetUserResponse:
+    user = bindings.v1User(
+        active=True,
+        admin=False,
+        username="determined",
```

Comment:
There's something about the username being hidden from the caller that makes me slightly uncomfortable. But maybe "sample" means "you don't get to pick, be happy with what I give you", so the caller shouldn't have any expectation on the username.

If you foresee some system where the username is important, then maybe this makes sense to make an argument. If not, then cool, this works for me.

Reply:
yeah, i thought this too. didn't anticipate username being important to any test callers, but it's probably a good idea to make these configurable. so i updated the test responses to accept `kwargs`.